### ISSUE TITLE
Cannot install non Rancher based Helm Charts with v2.2.3-rc4

### ISSUE THREAD

**Issue by akkie** [rancher/rancher#19890]
**What kind of request is this (question/bug/enhancement/feature request):**
This is a bug

**Steps to reproduce (least amount of steps as possible):**
Try to install a chart from the Helm catalog

**Result:**
A JS error:
```js
Uncaught TypeError: Cannot read property 'forEach' of undefined
    at n.parseQuestionsAndAnswers (component.js:288)
    at n.initTemplateModel (component.js:266)
    at component.js:70
    at e.invoke (0:34688)
    at e.flush (0:34604)
    at e.flush (0:34766)
    at e._end (0:35237)
    at MutationObserver._boundAutorunEnd (0:34934)
```

**Other details that may be helpful:**
It tries to iterate over the template questions, which are Rancher specific.
```js
template.questions.forEach(function (q) {
        questions.push(q);
        var subquestions = Ember.get(q, 'subquestions');

        if (subquestions) {
          questions.pushObjects(subquestions);
        }
      });
```

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.2.3-rc4
- Installation option (single install/HA): single install

**Comment by alena1108** [comment]
@akkie did the same use case work for you on the previous rancher builds?

**Comment by akkie** [comment]
@alena1108 Our last version was 2.1.6. And in this version it has worked.

**Comment by LennardWesterveld** [comment]
Same issue on the latest v2.2.3-rc5 release, seems only appear when there are no rancher questions form (anwers.yaml).

**Comment by loganhz** [comment]
Thanks for the report! It will be fixed in 2.2.3

**Comment by westlywright** [comment]
Available in master and `2.2.55`

**Comment by davidnuzik** [comment]
Version: Master (v2.3) (5/7/19)

I can now install non-Rancher helm chart apps without any issues. I encounter no issues in the browser dev console.
